fetcher/dockerhub.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+#
+# ** License **
+#
+# Home: http://resin.io
+#
+# Author: Andrei Gherzan <andrei@resin.io>
+#
+
+import os
+import logging
+from docker import Client
+from fetcher.tar import tarFetcher
+from modules.util import *
+
+log = logging.getLogger(__name__)
+
+class dockerhubFetcher(tarFetcher):
+
+    def __init__ (self, conffile, version, remote):
```

Comment:
I wanted to change this at the end but then I was to lazy to do it. I will have it. Given that this state was tested as it is, would it be fine to have a top commit for this?